README.md

```diff
@@ -64,13 +64,28 @@ taken to be empty.
 ```julia
 cell3 = Cell([3.0; 2.0; 5.0])
 cell4 = Cell([4.0; 6.0])
+population  = construct(Population, deepcopy([cell1, cell3, cell4]))
 population2 = construct(Population, deepcopy([cell1, cell3, cell4]))
 population3 = construct(Population, deepcopy([cell1, cell3, cell4]))
 tissue1 = construct(Tissue, deepcopy([population, population2, population3])) # Make a Tissue from Populations
 tissue2 = construct(Tissue, deepcopy([population2, population, population3]))
 embryo = construct(Embryo, deepcopy([tissue1, tissue2])) # Make an embryo from Tissues
 ```
 
+## Human readable printing of the embryo structure
+```julia
+printHumanReadable(embryo)
```

Comment:
this function doesn't exist: I assume you meant to add it in the PR?

Reply:
Note that Julia doesn't use camel case for functions, so it should be `print_human_readable`